extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -686,40 +739,46 @@ export class AmpA4A extends AMP.BaseElement {
     const jwkSetPromises = this.getSigningServiceNames().map(serviceName => {
       dev().assert(getMode().localDev || !endsWith(serviceName, '-dev'));
       const url = signingServerURLs[serviceName];
+      const currServiceName = serviceName;
```

Comment:
Why rename `serviceName` here?  Why not just make it `currServiceName` in the loop index?